pkg/containerutil/cp_resolve_other.go

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
If one day we support this on Windows - we will need the win version of volumeNameLen